R/vignettes.r

```diff
@@ -2,26 +2,33 @@
 #'
 #' Builds package vignettes using the same algorithm that \code{R CMD build}
 #' does. This means including non-Sweave vignettes, using makefiles (if
-#' present), and copying over extra files. You need to ensure that these
-#' files are not included in the built package - ideally they should not
-#' be checked into source, or at least excluded with \code{.Rbuildignore}
+#' present), and copying over extra files. The files are copied in the 'doc'
+#' directory and an vignette index is created in 'Meta/vignette.rds', as they
+#' would be in a built package. 'doc' and 'Meta' are added to
+#' \code{.Rbuildignore}, so will not be included in the built package. These
+#' files can be checked into version control, so they can be viewed with
+#' \code{browseVignettes()} and \code{vignette()} if the package has been
+#' loaded with \code{load_all()} without needing to re-build them locally.
 #'
 #' @param pkg package description, can be path or package name.  See
 #'   \code{\link{as.package}} for more information
 #' @param quiet If \code{TRUE}, suppresses most output. Set to \code{FALSE}
 #'   if you need to debug.
+#' @param install If \code{TRUE}, install the package before building
+#'   vignettes.
```

Comment:
This installs current dev version to a temporary library that is used just to render the vignette, right? I believe this happens elsewhere too now (README?), so it would be good to nail the wording and use in all places. Specifically, IIUC, the installation referred to here does not affect one's normal library and has no persistent effect. It makes the dev state of vignettes and README more parallel to docs below `man/`.

Reply:
Yes, that is right. I added this option mainly because it was taking a while to install the package when I was writing / testing this. We could alternatively just remove the option...

Reply:
Either way, need to describe the temporary install in the docs.